lua/lspconfig/configs.lua

```diff
@@ -84,7 +88,7 @@ function configs.__newindex(t, config_name, config_def)
     end
 
     -- Used by :LspInfo
-    M.get_root_dir = config.root_dir
+    M.get_root_dir = get_root_dir
```

Comment:
This needs to be reverted right?

Reply:
I think it was a bug in the code tbh. `get_config_dir`  is used in the code, but we expose `config.root_dir`. Probably best to expose `get_root_dir` instead? 

Based on the current code, you'll get the same result, so doesn't really matter. Thought I'd change it for consistency

Reply:
Ah, that's fine. can you put it under it's own commit though with the message?

Reply:
done